packages/labs/task/src/task.ts

```diff
@@ -134,7 +176,7 @@ export class Task<
   /**
    * Controls if they task will run when its arguments change. Defaults to true.
```

Comment:
Mention what `'afterUpdate'` means. Who should use it and when? Is it a temporary migration aid?

Reply:
That's in the options description. I'll copy down.